modules/2_nbde/files/powervs-setup.yml

```diff
@@ -26,28 +27,31 @@
     - name: Setup Proxy Info
       become_user: root
       shell: |
-        echo "Setting up proxy details..."
```

Comment:
@gauravpbankar @aditijadhav38 here is the guard

Reply:
ok